lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -394,15 +394,12 @@ sub infopanel {
     $self->render('test/infopanel');
 }
 
-sub get_current_job ($self, $with_assets = 0) {
-    return $self->reply->not_found unless defined $self->param('testid');
-
-    my $job = $self->schema->resultset("Jobs")
-      ->find($self->param('testid'), {$with_assets ? (prefetch => qw(jobs_assets)) : ()});
-    return $job;
+sub _get_current_job ($self, $with_assets = 0) {
+    die 'Missing testid' unless $self->param('testid');
```

Comment:
This will result in a server error. I don't think we want that.

Reply:
Yes, we want a 400 response, not 500 and an error in the server logs.